UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/NMDataService.cs

```diff
@@ -190,11 +192,15 @@ public async Task<List<ShowDailyFilesResponseModel>> GetDailyBatchDetailsFiles(s
         {
             try
             {
-                string accessToken = await _authFssTokenProvider.GenerateADAccessToken(correlationId);
+                string accessToken = await _authFssTokenProvider.GenerateADAccessToken(_userService.IsDistributorUser, correlationId);
 
-                _logger.LogInformation(EventIds.ShowDailyFilesResponseStarted.ToEventId(), "Maritime safety information request to get daily NM files response started with _X-Correlation-ID:{correlationId}", correlationId);
+                _logger.LogInformation(EventIds.ShowDailyFilesResponseStarted.ToEventId(), "Maritime safety information request to get daily NM files response started for daily user:{SignInName} and Identity:{userId} with _X-Correlation-ID:{correlationId}", _userService.SignInName ?? "Public", _userService.UserIdentifier, correlationId);
 
-                const string searchText = $" and $batch(Frequency) eq 'daily'";
+                string searchText = $" and $batch(Frequency) eq 'Daily' and $batch(Content) eq null ";
+                if (_userService.IsDistributorUser)
+                {
+                    searchText = $" and $batch(Frequency) eq 'Daily' and $batch(Content) eq 'Tracings' ";
```

Comment:
There is no unit test that covers this path.  Tests need to be extended to check that the right search text is being provided to `IFileShareService.FSSBatchSearchAsync` please.

Reply:
Thanks for highlighting. Added unit test to cover search query path for distributor.